src/arithmetic.jl

```diff
@@ -34,6 +33,145 @@ function ==(a::HomogeneousPolynomial, b::HomogeneousPolynomial)
     return iszero(a.coeffs) && iszero(b.coeffs)
 end
 
+# Iss: #209
```

Comment:
I don't think including an issue number is valuable. `## Inequality ##` would be better, in line with the above.

Reply:
I added the comment `## Total ordering ##`.